lib/streamlit/runtime/credentials.py

```diff
@@ -102,6 +106,57 @@ def email_prompt() -> str:
 }
 
 
+def _send_email(email: str) -> None:
+    """Send the user's email to segment.io, if submitted"""
+
+    if email is None or "@" not in email:
+        LOGGER.error("That doesn't look like an email :(")
+        return
+
+    headers = {
+        "authority": "api.segment.io",
+        "accept": "*/*",
+        "accept-language": "en-US,en;q=0.9",
+        "content-type": "application/json",
+        "origin": "http://localhost:8501",
+        "referer": "http://localhost:8501/",
+        "sec-fetch-mode": "cors",
+        "sec-fetch-site": "cross-site",
+        "user-agent": "",
+    }
+
+    dt = datetime.utcnow().isoformat() + "Z"
+
+    ANONYMOUS_ID = "cf41d0b9-8977-409a-852e-de16f281160e"
```

Comment:
nit: What does this `ANONYMOUS_ID` refer to? Maybe you can add a short comment here.

Reply:
I thought I couldn't submit `null`, but that turns out not to be true